docs/protocol-extension.md

```diff
@@ -1,6 +1,8 @@
 # GraphQL over WebSocket Protocol extension
 
-The GraphQL over WebSocket Protocol used by `mercurius` follows apollo's protocol defined [here](https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md). In order to achieve gateway-to-service communication and handle `connection_init` per client, an extension is used on the protocol for server-to-server communication.
+The GraphQL over WebSocket Protocol `mercurius` uses by default is [graphql-transport-ws](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md). It also supports the protocol used by Apollo's `subscriptions-transport-ws`; however that library is **unmantained** so it's not recommended to use.
```

Comment:
If possible, I'd deprecate the old graphql-ws protocol in the next major, seems not worth to keep supporting it because the `subscriptions-transport-ws` states it's unmaintained.